src/kernel/logging.c

```diff
@@ -74,15 +74,16 @@ void logline(log_level_t level, const char* msg){
                     fbCurrentLine++;
                 }
                 else {
-                    _fb_printStr(logLevelStrings[level], 0, fbCurrentLine, 0xFFFFFFFF, 0);
-                    _fb_printStr(msg, logLevelStrLen, fbCurrentLine, 0xFFFFFFFF, 0);
+                    //TODO: fbCurrentLine should be aligned with cur_fbLine in the framebuffer case.
```

Comment:
hmm yeah, this is an interesting one - I remember doing that because there no state tracking a current line number for the terminal output.
Now that the framebuffer keeps track of that itself, `fbCurrentLine` can be removed I reckon, and just use the new codepath. Log messages will work nicely with other things that print directly to the framebuffer now too.